src/FluentCMS.Application/Sites/RemoveUrlSiteCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MediatR;
+
+namespace FluentCMS.Application.Sites;
+
+public class RemoveUrlSiteCommand : IRequest
```

Comment:
I think these names are better for this command:
- RemoveUrlFromSiteCommand
- RemoveSiteUrlCommand

I prefer `RemoveSiteUrlCommand`